sources/packages/web/src/components/common/OfferingForm.vue

```diff
@@ -1,6 +1,7 @@
 <template>
+  <!-- todo: ann form definition -->
```

Comment:
Add form definition?

Reply:
as in my PR description and dev chat. it will be in my next PR. 
![image](https://user-images.githubusercontent.com/77353155/188738679-017ff91f-a023-4e62-8647-ed01c69d2c0c.png)